TV by the Numbers.js

```diff
@@ -1,144 +1,208 @@
 	(at your option) any later version.
 
-	This program is distributed in the hope that it will be useful,
+	Zotero is distributed in the hope that it will be useful,
 	but WITHOUT ANY WARRANTY; without even the implied warranty of
-	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-	GNU General Public License for more details.
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
 
-	You should have received a copy of the GNU General Public License
-	along with this program.  If not, see <http://www.gnu.org/licenses/>.
-*/
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
 
-/*
-	This translator supports saving a snapshot of a single post and saving
-	the citation of many posts at once without visiting each post. Thus, it does
-	not save a snapshot when multiple citations are to be saved.
+	***** END LICENSE BLOCK *****
 */
 
-
-PUB_TITLE = "TV by the Numbers";
-XPATH_TITLE = "//title";
-XPATH_PAGES = null;
-XPATH_DATE = "substring-after(substring-before(string(//p[@class='posted_on']),' by '), 'on ')";
-RE_DATE = /(.*)/;
-XPATH_AUTHORS = "substring-after(string(//p[@class='posted_on']),' by ')";
-RE_AUTHORS = /(.*)/;
-
 function detectWeb(doc, url) {
-	/* site has lots of garbage, check we're on the right doc */
-	if (!xpath_string(doc, doc, XPATH_TITLE))
-		return;
-	var posts = doc.evaluate("count(//div[@class='post-alt blog'])", doc, null,
-		XPathResult.NUMBER_TYPE, null).numberValue;
-	if (posts  == 1)
-		return "webpage";
-	else if (posts > 1)
+	if (url.indexOf('/?s=') > -1 && getSearchResults(doc, true))
 		return "multiple";
+	else if (doc.body.className.indexOf("single-post") > -1)
+		return "webpage";
 }
 
-function xpath_string(doc, node, xpath) {
-	var res = doc.evaluate(xpath, node, null, XPathResult.STRING_TYPE, null);
-	if (!res || !res.stringValue)
-		return null;
-	return Zotero.Utilities.trim(res.stringValue);
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function(items) {
+			if (!items) {
+				return true;
+			}
+			var articles = [];
+			for (var i in items) {
+				articles.push(i);
+			}
+			ZU.processDocuments(articles, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = ZU.xpath(doc, '//div[@class="container container-small"]/article/h2/a');
+	for (var i = 0; i < rows.length; i++) {
+		var href = rows[i].href;
+		var title = ZU.trimInternal(rows[i].textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
 }
 
-function xpre(doc, node, xpath, reg) {
-	var xpmatch = xpath_string(doc, node, xpath);
-	return reg.exec(xpmatch)[1];
+
+
+function doWeb(doc, url) {
```

Comment:
You already have the function `doWeb` above, i.e. this is duplicate content and should be deleted.